_posts/2017-03-17-cwl-hackathon.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "CWL week in London"
+date:   2017-03-17 18:44:27 +0300
+comments: true
+---
+After the successful completion of my GSoC projects, I had been invited to meet my mentors Michael R. Crusoe and Roman Valls Guimera on the CWL work session that took place in London from the 1st to the 4th November 2016. I was thrilled by this opportunity, and though it took quite a while for me to organize this journey (my first single voyage abroad), it was undoubtfully worth it. 
```

Comment:
Line wrapping at ~ 80 columns would make this easier to review :-)

'... for **the** CWL work session'

' 1st to the 4th November 2016.' -> ' 1st to the 4th of November, 2016.

'(my first single voyage abroad)' - '(my first solo voyage abroad)